nature.md

```diff
@@ -0,0 +1,1441 @@
+- **Not Evaluated**: The product has not been evaluated against the criterion. This can be used only in WCAG 2.0 Level AAA.
+
+## WCAG 2.0 Report
+
+Tables 1 and 2 also document conformance with:
+- EN 301 549: Chapter 9 - Web, Chapter 10 - Non-Web documents, Section 11.2.1- Non-Web Software (excluding closed functionality), and Section 11.2.2 - Non-Web Software (closed functionality).
+- Revised Section 508: Chapter 5 – 501.1 Scope, 504.2 Content Creation or Editing, and Chapter 6 – 602.3 Electronic Support Documentation.
+
+**Note**: When reporting on conformance with the WCAG 2.0 Success Criteria, they are scoped for full pages, complete processes, and accessibility-supported ways of using technology as documented in the [WCAG 2.0 Conformance Requirements](https://www.w3.org/TR/WCAG20/#conformance-reqs).
+
+### Table 1: Success Criteria, Level A
+
+<table>
+<thead>
+<tr>
+<th><strong>Criteria</strong></th>
+<th><strong>Conformance Level </strong></th>
+<th><strong>Remarks and Explanations</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td><a href="http://www.w3.org/TR/WCAG20/#text-equiv-all"><strong>1.1.1 Non-text Content</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.1 (Web)</li>
+<li>10.2.1 (non-web document)</li>
+<li>11.2.1.1 (Software)</li>
+<li>11.2.2.1 (Closed Functionality Software)</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+With rare exceptions, most images that we supply provide alternative text, or descriptions, or figure captions. We may use WAI-ARIA techniques to link some complex images with their descriptions if alternative text is unable to provide adequate context. We are working towards full support.  
+<br><br>
+Background images such as icons on user interface elements are generally identified with text labels. These may be visible, visually-hidden, or supplied using WAI-ARIA techniques.
+<br><br>
+We mark up some images so that they can be ignored by Assistive Technologies if they are <a href="https://www.w3.org/TR/WCAG21/#dfn-pure-decoration"><i>pure decoration</i></a> (as defined by WCAG 2.0).
+<br><br>
+The Nature subscription pages at <a href="https://www.nature.com/nature/subscribe">https://www.nature.com/nature/subscribe</a> currently contain SVG images without text alternatives, and decorative SVG images that are not marked up to be ignored by Assistive Technologies. We are actively working to repair these problems. 
+<br><br>
+Third party scripts may inject non-text content without text alternatives. These include images in advertising scripts, in Disqus commenting widgets, and in the "manage cookies" privacy centre. Where we know of deficiencies, we have contacted third party suppliers to request repairs.
```

Comment:
You specifically mention Disqus here.  Should we also specifically mention OneTrust as the manage cookies thing?